test/parallel/test-buffer-fakes.js

```diff
@@ -41,14 +41,6 @@ assert.throws(function() {
   fb.indexOf(5);
 }, TypeError);
 
-assert.throws(function() {
-  fb.readFloatLE(0);
-}, TypeError);
-
-assert.throws(function() {
-  fb.writeFloatLE(0);
-}, TypeError);
-
```

Comment:
Note: this does not throw anymore and works the same as any other regular buffer method from now on. This was only important for the c++ code that now does not exist anymore.